### ISSUE TITLE
queue size gets reset to default after change in client

### ISSUE THREAD

**Issue by shawn8888** [haugene/docker-transmission-openvpn#1498]
**Describe the problem**

I upgrade from 2.x to 3.x recently, and I noticed this bug in 3.x.
I use Transmission Remote GUI in Windows as a client tool to connect to the container. The container's environment variables have:
```
TRANSMISSION_DOWNLOAD_QUEUE_ENABLED = false
TRANSMISSION_DOWNLOAD_QUEUE_SIZE = 20

```
After I start the container, I open the client tool to set the queue size and enable the queue control. It controls the queue size until the next time I restart the container and that's how it works in 2.x version.

However, after version 3.x, I set the queue size in client, but the queue settings often get reset to the default for no reason after a few hours running without restart. 

--------------------------------------------

Another bug is the port forwarding stops working after a few minutes or a few hours. I don't know if the two are related. I use PIA and CA Montreal

--------------------------------------------
Update: the port forwarding start working again without a restart. So I guess it's just not stable, and when it stopped, some background jobs can bring it back in a few minutes.

Please help.

**Comment by haugene** [comment]
Sounds very strange that configs reset themselves without a restart. But just remove `TRANSMISSION_DOWNLOAD_QUEUE_ENABLED` and `TRANSMISSION_DOWNLOAD_QUEUE_SIZE` if you're going to set them in the web. In 3.x they don't get overridden if the env vars are not set. Of course given that you have mounted /data to your disk and have not overridden transmission_home (then you need to mount that).

About the port forwarding and server instability. There are a bunch of reported struggles with the new PIA servers. The CA servers particularly if my memory serves right. See #1334. Different servers are working at different times and some are giving auth_failed where others allow you access with the same credentials :man_shrugging:

**Comment by shawn8888** [comment]
Thank you haugene.
I use the pxxxxxx password. I found the port forwarding is unstable, especially on CA Toronto. After switching to CA Montreal, no more crashes, but port forwarding sometimes fail, and sometimes it can recover by itself. Not sure why the other two servers,Ontario and Vancouver are not supported? 

I like to put all my favorite settings in container Env Vars, so I don't have to touch them every time I restart it. But I do change the queue size a lot and I like to keep it between two restarts. 
I have removed the two download queue env vars, and restarted the container. I will keep it running and see if the reset issue goes away.
